src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/LogsController.kt

```diff
@@ -0,0 +1,24 @@
+package org.vaccineimpact.orderlyweb.controllers.web
+
+import org.vaccineimpact.orderlyweb.ActionContext
+import org.vaccineimpact.orderlyweb.controllers.Controller
+import org.vaccineimpact.orderlyweb.db.repositories.OrderlyWebReportRunRepository
+import org.vaccineimpact.orderlyweb.db.repositories.ReportRunRepository
+import org.vaccineimpact.orderlyweb.models.RunningReports
+
+class LogsController(
```

Comment:
We discussed this in passing in relation to Lekan's question - I think this class should be called `ReportRunController`, analogous to the existing `api.ReportRunController`. It is the home for code used by both the web interface and the API, whereas this one is just for the web.